Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "Clp"
+uuid = "5e71592d-7078-450d-8580-c595f51aa13b"
+authors = ["Mathieu Besançon <mathieu.besancon@gmail.com>"]
```

Comment:
I would not include this field for consistency with other solver repo in JuliaOpt.

Reply:
sorry hadn't seen it was still there, it got generated